### PR TITLE
Added emojis POG

### PR DESCRIPTION
**Pull request by MrMugame**

### CODE REVIEW

**Comment by wubbl0rz** [comment]
KEKWait

File: README.md [hunk 1]
```diff
@@ -70,7 +70,7 @@ Short version:
 - Refresh Page. (F5)
 - Store should show this repo now and you can install the addon.
 
-## Options / Usage
+## Options ⚙️/ Usage 📈
```
Comment (wubbl0rz) [inline README.md:73]: pls remove graph emoji

**Review by wubbl0rz — CHANGES_REQUESTED** [review]

**Comment by wubbl0rz** [comment]
1x rebase pls

File: README.md (hunk 1 above)
Comment (MrMugame) [reply README.md]: [a4c85d1](https://github.com/wubbl0rz/FiatChamp/pull/26/commits/a4c85d17a0d7c8b6f9ab9bc6c2398ccf5d97fe72)

**Pull request merged**